### PR TITLE
chore(github-growth): set commitfilechange language

### PR DESCRIPTION
**Pull request by cathteng**
When we process a Github commit webhook and create `CommitFileChange` objects for files modified in a commit, if a file has an extension in the `EXTENSION_LANGUAGE_MAP` (created with [this script](https://gist.github.com/cathteng/9949a6b8034f60be5e0802f0dfb4f7d6)), then set the language field in the `CommitFileChange` object.

For ER-1799

### CODE REVIEW

**Review by scefali — APPROVED** [review]

File: src/sentry/constants.py [hunk 1]
```diff
@@ -700,3 +700,205 @@ def from_str(cls, string: str) -> Optional[int]:
     "*/healthz",
     "*/ping",
 ]
+
+# Generated from https://raw.githubusercontent.com/github-linguist/linguist/master/lib/linguist/languages.yml and our list of platforms/languages
```
Comment (JoshFerge) [inline src/sentry/constants.py:704]: might be worth including this script in the repository in the `/bin`/ folder

File: src/sentry/integrations/github/webhook.py [hunk 2]
```diff
@@ -357,16 +357,31 @@ def _handle(
                         date_added=parse_date(commit["timestamp"]).astimezone(timezone.utc),
                     )
                     for fname in commit["added"]:
+                        language = EXTENSION_LANGUAGE_MAP.get(fname.split(".")[-1])
```
Comment (JoshFerge) [inline src/sentry/integrations/github/webhook.py:360]: what happens if the file name doesn't have an extension?

Comment (cathteng) [reply src/sentry/integrations/github/webhook.py:360]: the field is set to None

Comment (JoshFerge) [reply src/sentry/integrations/github/webhook.py:360]: what happens if the file name is the name of an extension? e.g. if i have a file called `tsx`, but its filled with nonsense, not typescript?

Comment (JoshFerge) [reply src/sentry/integrations/github/webhook.py:360]: (just thinking maybe we should validate that an extension exists before trying to use `EXTENSION_LANGUAGE_MAP`

Comment (cathteng) [reply src/sentry/integrations/github/webhook.py:360]: i see. if you do `"tsx".split(".")[-1]` you get `"tsx"` back which wouldn't be correct

**Pull request merged**